exec/fixedosc_llhscan.cc

```diff
@@ -246,7 +251,7 @@ void fixedosc_llhscan(const std::string &fitConfigFile_,
           }
         }
         // oscDist is just the reactor PDF for this point in the oscillation scan, so we scale that by the expected reactor rate
-        oscDist.Scale(noms[it->first]);
+        oscDist.Scale(noms_config * ratio);
```

Comment:
So if this would be just `noms[it->first]` if line ~227 was `noms[it->first] = noms[it->first] * ratio;`

Reply:
I think ignore this comment after what we discussed in person